test/system/app-enablement_test.go

```diff
@@ -0,0 +1,349 @@
+/*
+ * Copyright (c) 2020  InterDigital Communications, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package systemTest
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"context"
+	"time"
+
+	log "github.com/InterDigitalInc/AdvantEDGE/go-packages/meep-logger"
+	smc "github.com/InterDigitalInc/AdvantEDGE/go-packages/meep-service-mgmt-client"
+        asc "github.com/InterDigitalInc/AdvantEDGE/go-packages/meep-app-support-client"
```

Comment:
Fix indentation & formatting for all files.

Reply:
didn't do a goimport on that new file.. now i did...